lib/web_ui/lib/src/engine/embedder.dart

```diff
@@ -124,6 +124,9 @@ class FlutterViewEmbedder {
   HostNode? get glassPaneShadow => _glassPaneShadow;
   HostNode? _glassPaneShadow;
 
+  DomElement? get textEditingHostNode => _textEditingHostNode;
+  DomElement? _textEditingHostNode;
```

Comment:
I was planning on removing the `HostNode` because the original version was only to decide if we were going to render in an Element or the ShadowDOM of an element.

We can repurpose the name for any other thing we like.

My idea was to have a FlutterApplicationElement (call it HostNode) that knows how to create all the different elements for a Flutter Web app view (the glass-pane, the different element hosts, etc...), and that can be used later in place of the global "textEditingNode", "glassPaneElement", "semanticsHelper" and family (because it's likely that the global ViewEmbedder needs to go away in a multi-view world)